markets/legacy-market/contracts/LegacyMarket.sol

```diff
@@ -277,16 +284,38 @@ contract LegacyMarket is ILegacyMarket, Ownable, UUPSImplementation, IMarket, IE
             debtValueMigrated
         );
 
+        if (v3System.isVaultLiquidatable(preferredPoolId, address(oldSynthetix))) {
+            revert Paused();
```

Comment:
this would be an extreme edge case. in the unlikely even that somehow the vault is in a liquidatable state (whether it was in that state before migration, or after the migration was completed), it means there is something horribly wrong on th ev3 side and it probably wouldn't even be possible to continue operations on v3, or there is some temporary bug or something.

in either case, I don't see any reason why this would be a problem to add this check since it gives us less to think bout if we found ourselves in such an apololyptic scenario.